### PR TITLE
Added toggle button to show information

### PR DESCRIPTION
**Pull request by hennifant**

### CODE REVIEW

File: src/components/FishCard.js [hunk 1]
```diff
@@ -1,40 +1,53 @@
+import { useState } from 'react';
 import styled from 'styled-components';
 
 export default function FishCard({ fish }) {
+  const [showDetails, setShowDetails] = useState(false);
   return (
     <FishContainer>
       <FishName>{fish.FishGerman}</FishName>
-      <PropertyList>
```
Comment (JelenadlS) [inline src/components/FishCard.js:7]: When I do see it right, you kicked out the ul and you are starting directly with li, right? Therefore you should keep the Propertylist😊

File: src/components/FishCard.spec.js [hunk 2]
```diff
@@ -23,7 +24,16 @@ describe('FishCard', () => {
     const fishName = screen.getByRole('heading', { level: 2 });
     expect(fishName).toHaveTextContent('Siamesischer Kampffisch');
 
+    const button = screen.getByRole('button', { name: /Weitere/i });
+    expect(button).toBeInTheDocument();
+
+    userEvent.click(button);
```
Comment (JelenadlS) [inline src/components/FishCard.spec.js:30]: When I am right, the userEvent.click is used in combination with a mock function in order to call it. Therefore both userEvents could be out here. However, you could test the toggle (if this is possible, im actually not sure  😅)

**Review by JelenadlS — CHANGES_REQUESTED** [review]
Just two little comments 😊
Overall looking well structured! 😊

File: src/components/FishCard.js (hunk 1 above)
Comment (hennifant) [reply src/components/FishCard.js:7]: Oh, das habe ich übersehen, danke für den Hinweis.

File: src/components/FishCard.spec.js (hunk 2 above)
Comment (hennifant) [reply src/components/FishCard.spec.js:30]: My thought was that I test if the button is available and at the first click the list is displayed, then at the second click if the list is no longer displayed.

Comment (JelenadlS) [reply src/components/FishCard.spec.js:30]: well, lets say i am not the best one in testing😅
however, i think in that case you need to call the onClick={toggleProperties} with a mock function such as:
const toggleCallback = jest.fn();
nClick={toggleCallback} 
...
    expect(toggleCallback).toHaveBeenCalledWith({
      ...oneObject...
    });

Just an idea - might also not work 😊

Comment (hennifant) [reply src/components/FishCard.spec.js:30]: thanks for the hint, I'll take a look at it - testing is also not my big strength - as you can see :-D

**Pull request merged**